README.md

```diff
@@ -117,6 +117,8 @@ cgltf also supports some glTF extensions:
 - KHR_materials_unlit
 - KHR_materials_variants
 - KHR_materials_volume
+- KHR_materials_iridescence
```

Comment:
nit: this list was sorted by name I believe.

Reply:
whoops, and looks like I duplicated the iridescence listing. Thanks for catching this.